src/workspace-tree/icons.ts

```diff
@@ -60,23 +59,19 @@ const SPECIFIC_RULE_CLASS_ICONS: Record<string, string> = {
  */
 export function getBazelRuleIcon(
   target: blaze_query.ITarget,
-): string | vscode.ThemeIcon {
+): IconName | undefined {
   const ruleClass = target.rule.ruleClass;
   let iconName = SPECIFIC_RULE_CLASS_ICONS[ruleClass];
   if (!iconName) {
     if (ruleClass.endsWith("_binary")) {
-      iconName = "binary";
+      iconName = IconName.BINARY;
```

Comment:
random idea: maybe we should make this lookup configurable? I assume many people have custom in-house Bazel rules, and still might want to have nice icons for those

Reply:
SG, and I'm also thinking about if we should try adopt [vscode-icons](https://github.com/vscode-icons/vscode-icons/wiki/ListOfFiles) in the future, but using same set of the icons may also confuse the users between "files view" and "workspace rules view"...